cmd/monaco/generate/deletefile/deletefile.go

```diff
@@ -185,6 +189,28 @@ func skipping(ttype string, included, excluded map[string]struct{}) bool {
 	return false
 }
 
+func toMapKey(d persistence.DeleteEntry) string {
```

Comment:
nit:  a short doc comment for the method would be nice so it's clear what it's for/why we build string keys and can't use the DeleteEntries directly.